receiver/kubeletstatsreceiver/metadata.yaml

```diff
@@ -500,6 +500,14 @@ metrics:
     gauge:
       value_type: int
     attributes: []
+  k8s.pod.volume.usage:
+    enabled: false
+    description: "The number of used bytes in the pod volume."
+    unit: By
+    sum:
+      value_type: int
+      monotonic:
```

Comment:
I think we can use https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/68952bf10786f2a38f436009221d3bbd21e8dc21/receiver/filestatsreceiver/templates/registry/metadata/metadata.yaml.j2#L33-L36 as guidance for this.